documentation/book/con-customizing-image-pull-policy.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-image-pull-policy-{context}']
+= Customizing Image Pull Policy
```

Comment:
```suggestion
= Customizing the image pull policy
```

Reply:
Feel free to ignore suggestions about the capitalization of 'Image Pull Policy' if I'm wrong here.